packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -147,10 +71,10 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "22.4.3",
+                        "desktop": "<=22.4.3",
                         "mobile": "!",
-                        "web": "22.4.3",
-                        "revision": "969161f41dcd889fa1fb0cf5e1d3437fddd8c26d"
+                        "web": "<=22.4.3",
```

Comment:
Wait, wait, you meant `>=22.4.3`, no?

Reply:
Yes. I spent about 1 minute to make this condition right and in the end, I messed it up 🤦

Reply:
Fixed